### DISCUSSION TITLE
Rustdesk installation with predefined data

### DISCUSSION THREAD

**Discussion by mikekgr** [rustdesk/rustdesk#865] [General]
Dear Sirs,
is there any easy way to integrate in the installation of rustdesk as predefined : the ID Server ,  Relay Server and Key ? I want this because I use Self-host Server.
Thanks in advance.

**Comment by supply9243** [comment]
Installation :: Documentation for RustDesk
https://rustdesk.com/docs/en/self-host/install/#step-3--set-hbbshbbr-address-on-client-side

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/769
